custom.example.css

```
@@ -89,23 +119,33 @@ body{}
 .navigation--item{}
+.navigation--item:hover{}
 .navigation--item--separator{}
 .navigation-hidden{}
 .navigation svg{}
+.rounded{}
 .navigation--flush-cache svg{}
 .navigation--fullscreen--exit{}
 .navigation--fullscreen-enabled .navigation--fullscreen--exit{}
 .navigation--fullscreen-enabled .navigation--fullscreen--enter{}
-.navigation--control--play{}
+.navigation--media-buttons{}
```

Biome

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)